packages/data-api/scripts/dropAnalyticsTableIndexes.sql

```diff
@@ -0,0 +1,16 @@
+do $$
+declare
+  tStartTime TIMESTAMP;
+
+begin
+  RAISE NOTICE 'Dropping analytics table indexes...';
+
+  tStartTime := clock_timestamp();
+  DROP INDEX IF EXISTS analytics_data_element_entity_date_idx;
+  RAISE NOTICE 'Dropped [data_element, entity, date] index, took %', clock_timestamp() - tStartTime;
+
+  tStartTime := clock_timestamp();
+  DROP INDEX IF EXISTS analytics_data_element_entity_data_group_event_date_idx;
```

Comment:
The name of this index (and the columns it uses) has been changed in: https://github.com/beyondessential/tupaia-backlog/issues/2714

That issue is currently part of `psss-phase-2` branch, and I believe won't be merged before this one, so I guess it's up to someone working on `psss-phase-2` to ensure that this script gets updated as well

Reply:
Good catch @rohan-bes! Very glad you reviewed this, because that would have been a soft conflict that may have gone unnoticed. I'll remember to let `psss-phase-2` folks know when this has hit dev for their next update

Reply:
One possible way we could make this more robust is remove the `IF EXISTS` from the `DROP` statement? I suppose that way it asserts that the index is there? However, that would make this script more annoying to use on a partially built / weird state analytics table so perhaps not... I'm fine with it as is 😄

Reply:
Yeah I see what you mean...It would mean we were more likely to catch changes to the index name, but also make the script a bit more constrained in its use. I'm on the fence!